_data/Pratibha-Goyal/Pratibha-Goyal.md

```diff
@@ -0,0 +1,5 @@
+name: Pratibha Goyal
+institution: LNM Institute of Information Technology
+profile_pic: pratibha.png  picture must be squared and 544px on width and height.
```

Comment:
comment out or remove the extra statements i.e `picture must be squared and 544px on width and height.`